src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form class="form">
+      <div class="personalInfo">
+        <fieldset class="allFieldset">
+          <legend class="allLegend">
+            Personal information
+          </legend>
+          <div class="personalInput">
+            <label class="personalDate">
+              Surname:
+              <input
+                type="text"
+                name="surname"
+                autocomplete="off"
+              >
+            </label>
+            <label class="personalDate">
+              Name:
+              <input
+                type="text"
+                name="name"
+                autocomplete="off"
+              >
+            </label>
+            <label class="personalDate">
+              How old are You?
+              <input
+                type="number"
+                name="age"
+                min="1"
+                max="100"
+                value="12"
+              >
+
+            </label>
+            <label class="personalDate">
+              Full date of birth:
+              <input type="date" name="fullAge">
+            </label>
+            <label class="acceptInformation">
+              I accept the terp of the agreement
+              <input
+                type="checkbox"
+                name="terms"
+                required
+              >
+            </label>
+          </div>
+        </fieldset>
+      </div>
+
+      <div class="registration">
+        <fieldset class="allFieldset">
+          <legend class="allLegend">
+            Registration
+          </legend>
+          <label class="email">
+            E-mail:
+            <input
+              required
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+            >
+          </label>
+          <label class="password">
+            Password:
+            <input
+              required
+              type="password"
+              name="password"
+              minlength="5"
+              maxlength="20"
+            >
+          </label>
+        </fieldset>
+      </div>
+
+      <div class="aboutYou">
+        <fieldset class="allFieldset">
+          <legend class="allLegend">
+            An interesting fact about you!
+          </legend>
+          <label class="loveCat">
+            Do you love cats?
+            <input type="radio" name="loveCat">
+            <label class="radioInput">
```

Comment:
how do you connect your input and label?

Reply:
I do not quite understand what you mean "how to connect"? do they have to connect somehow?